From 62743c6c759d137ff324187ec53c40cf7aaf70d4 Mon Sep 17 00:00:00 2001
From: Patrik Flykt <patrik.flykt@intel.com>
Date: Wed, 6 Oct 2021 16:36:21 +0300
Subject: [PATCH 2/3] qatlib: Remove remaining -shared flags

Remove remaining -shared flags from the build so that static
libraries are also built.

Signed-off-by: Patrik Flykt <patrik.flykt@intel.com>
---
 Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 4565d85..24ade3a 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -114,7 +114,6 @@ lib@LIBUSDMNAME@_la_SOURCES = \
 lib@LIBUSDMNAME@_la_CFLAGS = -I$(srcdir)/quickassist/utilities/libusdm_drv \
 			-I$(srcdir)/quickassist/utilities/libusdm_drv/include \
 			-I$(srcdir)/quickassist/utilities/libusdm_drv/user_space \
-			-shared \
 			$(COMMON_FLAGS)
 lib@LIBUSDMNAME@_la_LDFLAGS = -version-info $(LIBUSDM_VERSION) \
 	$(COMMON_LDFLAGS) \
@@ -219,7 +218,6 @@ lib@LIBQATNAME@_la_CFLAGS = -I$(srcdir)/quickassist/utilities/libusdm_drv \
 		  -I$(srcdir)/quickassist/lookaside/access_layer/src/qat_direct/vfio \
 		  -D USER_SPACE \
 		  -D LAC_BYTE_ORDER=__LITTLE_ENDIAN \
-		  -shared \
 		  $(COMMON_FLAGS)
 lib@LIBQATNAME@_la_LIBADD = libosal.la libadf.la lib@LIBUSDMNAME@.la -lcrypto
 lib@LIBQATNAME@_la_LDFLAGS = -version-info $(LIBQAT_VERSION) \
-- 
2.36.1

